### PR TITLE
Fixes #199

### LINKED ISSUE #199: The back button and language icon overlap each other on mobile
This bug is on the `languages/:language`

### PR DESCRIPTION
**Pull request by sonakshisaxena1**
Removes the overlapping of the back button and sidebar icon on mobile screens. 
The website will look in the following way  - 

edit

### CODE REVIEW

File: src/components/Dashboard/index.jsx [hunk 1]
```diff
@@ -68,6 +70,9 @@ import Sidebar from '../../components/Sidebar';
       fill: theme.palette.common.white,
     },
   },
+  leftMargin: {
+    marginLeft: leftMargin,
```
Comment (fiennyangeln) [inline src/components/Dashboard/index.jsx]: how about using 
```
position: absolute,
right: theme.spacing.unit,
``` 
?

Comment (sonakshisaxena1) [reply src/components/Dashboard/index.jsx]: Seems better!
I'll make the changes :)

**Comment by sonakshisaxena1** [comment]
@fiennyangeln I have made the required changes. Please review it once.

**Comment by fiennyangeln** [comment]
@sonakshisaxena1 LGTM! 👍 lets wait for Hassan opinion on this before we merge

**Comment by djmitche** [comment]
@fiennyangeln do you want to change your review to green?

File: src/components/Dashboard/index.jsx [hunk 2]
```diff
@@ -68,6 +68,10 @@ import Sidebar from '../../components/Sidebar';
       fill: theme.palette.common.white,
     },
   },
+  leftMargin: {
```
Comment (helfi92) [inline src/components/Dashboard/index.jsx]: Rename to `menuIconButton`.

File: src/components/Dashboard/index.jsx [hunk 3]
```diff
@@ -158,7 +162,9 @@ export default class Dashboard extends Component {
           {withSidebar && (
             <Hidden mdUp>
               <IconButton
-                className={classes.link}
+                className={classNames(classes.link, {
+                  [classes.leftMargin]: !header,
```
Comment (helfi92) [inline src/components/Dashboard/index.jsx]: `classes.menuIconButton`.

Comment (sonakshisaxena1) [reply src/components/Dashboard/index.jsx]: @helfi92  I have made the requested changes, please review it.

Comment (helfi92) [reply src/components/Dashboard/index.jsx]: @sonakshisaxena1 I don't see the new changes. Did you make sure to push your changes to this branch?

**Pull request merged**